client.py

```diff
@@ -84,30 +84,30 @@ def parse_ServerResponse(self, response):
 if args.verbose:
     print(res1)
 
-session_id = res1.getroot().attrib['SessionID']
+session_id = res1.attrib['SessionID']
 k = res1.find('.//{http://www.w3.org/2000/09/xmldsig#}RSAKeyValue')
 mod = number.bytes_to_long(d64sb(k.find(
   '{http://www.w3.org/2000/09/xmldsig#}Modulus').text))
 exp = number.bytes_to_long(d64sb(k.find(
   '{http://www.w3.org/2000/09/xmldsig#}Exponent').text))
-pubk = RSA.construct({'n': mod, 'e': exp})
+pubk = RSA.construct( (int(mod), int(exp)) )
```

Comment:
Yeah, I needed this too.

Reply:
Thanks, do you know in what version of PyCryptoDome this changed?

https://pycryptodome.readthedocs.io/en/latest/src/public_key/rsa.html#Crypto.PublicKey.RSA.construct